gapic-common/lib/gapic/rest/client_stub.rb

```diff
@@ -127,8 +141,11 @@ def make_put_request uri:, body: nil, params: {}, options: {}
       # @param params [Hash] query string parameters for the request
       # @param options [::Gapic::CallOptions,Hash] gapic options to be applied
       #     to the REST call. Currently only timeout and headers are supported.
+      # @param is_server_streaming [Bool] flag if method is streaming
```

Comment:
"is server setreaming" in the param description

Reply:
Done